orchestrator/index.js

```diff
@@ -0,0 +1,67 @@
+const request = require('request-promise');
+const HABANERO = require('node-red-habanero');
+
+const HABANERO_ADMIN_ROOT = '/orchestrator';
+const HABANERO_NODE_ROOT  = '/orchestrator-api';
+
+function init(server, app){
+	// Habanero
+	var habaneroSettings = require('../node_modules/node-red-habanero/settings');
+	habaneroSettings.httpAdminRoot = HABANERO_ADMIN_ROOT;
+	habaneroSettings.httpNodeRoot = HABANERO_NODE_ROOT;
+	// Initialise the runtime with a server and settings
+	HABANERO.init(server, habaneroSettings);
+	// Serve the editor UI
+	app.use(habaneroSettings.httpAdminRoot, HABANERO.httpAdmin);
+	// Serve the http nodes UI
+	app.use(habaneroSettings.httpNodeRoot, HABANERO.httpNode);
+	// auto-login url to habanero
+	app.get('/gotoHabanero', gotoHabanero);
+	HABANERO.start();
+}
+
+function getHabaneroBaseUrl(req){
+	var baseUrl = req.protocol + '://' + req.get('host') + HABANERO_ADMIN_ROOT +'/';
+	return baseUrl;
+}
+
+function doHabaneroLogin(habanero_base_url) {
+	return request({
+      	uri: habanero_base_url + 'auth/token',
+		method: 'POST',
+		form: {
+		  FROM_CONCARIA: 'true',
+		  client_id: "node-red-editor",
+		  grant_type: "password",
+		  scope:"",
```

Comment:
Use single quotes, but the linter will catch this as soon as you rebase on `development`.